test/hotspot/jtreg/runtime/ClassUnload/KeepAliveClass.java

```diff
@@ -71,11 +72,7 @@ public static void main(String... args) throws Exception {
     escape = null;
     ClassUnloadCommon.triggerUnloading();
 
-    {
-        boolean isAlive = wb.isClassAlive(className);
-        System.out.println("testClass (3) alive: " + isAlive);
-        ClassUnloadCommon.failIf(isAlive, "should be unloaded");
-    }
-
+    Set<String> aliveClasses = ClassUnloadCommon.triggerUnloading(List.of(className));
```

Comment:
The call to `triggerUnloading` at line 73 is redundant because you don't check the result afterwards and the call here will repeat it.

Reply:
oops missed one.